app/src/main/java/me/chester/minitruco/android/MesaView.java

```diff
@@ -39,6 +39,10 @@ public class MesaView extends View {
 
 	private int posicaoVez;
 
+	private int valorMao;
+
+	private final float density = getResources().getDisplayMetrics().density;
```

Comment:
Assumindo que a densidade do display não muda... mas tudo é possível nesse ecossistema, não vou me espantar se algum device for insano nesse nível 🤣